src/sys/power.rs

```diff
@@ -0,0 +1,303 @@
+use crate::sys::kernel::SceUid;
+
+//Power callback flags
+#[repr(u32)]
+pub enum Callback{
+    //Indicates the power switch it pushed, putting the unit into suspend mode
+    PowerSwitch     = 0x80000000,
+    
+    //Indicates the hold switch is on
+    HoldSwitch      = 0x40000000,
+
+    //Indicates the PSP has gone to standby (screen off)
+    Standby		    = 0x00080000,
+
+    //Indicates the resume process is complete. (Only triggered when another even happens)
```

Comment:
Nitpick: All of the comments in this file should be started with 3 slashes: `///`, this means they are included in the generated documentation, unlike `//`.

Reply:
Changes have been made